tests/util.py

```diff
@@ -5,6 +5,18 @@
 import urllib3
 
 
+def is_url_match(url1: str, url2: str) -> bool:
+    '''Checks if two urls point to the same resource'''
+    parsed1 = urllib3.util.parse_url(url1)
+    parsed2 = urllib3.util.parse_url(url2)
+    return (parsed1.host == parsed2.host
+            and parsed1.port == parsed2.port
+            and parsed1.path == parsed2.path
+            and (parsed1.query == parsed2.query
```

Comment:
I think you can do

query1 = parse_qs(parsed1.query)
query2 = parse_qs(parsed2.query)
    
and then just compare the query objects:
query1 == query2

please add a test for the method though.

Reply:
I don't know what function you mean. It is not available in urllib3. Maybe it is from the older urllib package but I won't include both versions.

[9235965](https://github.com/geo-engine/geoengine-python/pull/187/commits/923596557d6bf89ccf3a42014535a797cf0ff1a4)

Reply:
I mean this function: from urllib.parse import parse_qs
and even in the urllib3 docs they use urllib.parse alongside urllib3 https://urllib3.readthedocs.io/en/stable/user-guide.html#query-parameters

Reply:
[2cd6df4](https://github.com/geo-engine/geoengine-python/pull/187/commits/2cd6df407b536b20a5fb6458ea144710a60bab6a)